packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -267,9 +267,13 @@ export class SpecBuilder {
     return this;
   }
 
-  drainQueue(): this {
+  drainQueue(options?: { expectUserCallback: boolean }): this {
```

Comment:
Can you comment this, including the default behavior (especially since it's different than writeAcks and failWrite)?

Reply:
Reworked in https://github.com/firebase/firebase-js-sdk/pull/678 (and merged into this PR).